src/components/modal/README.md

```diff
@@ -587,6 +687,185 @@ component. This will hide the modal before another modal is shown.
+            // an error occured
+          })
+      },
+      showMsgBoxTwo() {
+        this.boxTwo = ''
+        this.$bvModal.msgBoxOk('Data was submitted successfully', {
+          title: 'Confirmation',
+          size: 'sm',
+          buttonSize: 'sm',
+          okVariant: 'success',
+          headerClass: 'p-2 border-bottom-0',
+          footerClass: 'p-2 border-top-0',
+          centered: true
+        })
+          .then(value => {
+            this.boxTwo = value
+          })
+          .catch(err => {
+            // an error occured
+          })
+      }
+    }
+  }
+</script>
+
+<!-- msg-box-ok.vue -->
+```
+
+### Confirm Message Box
+
+Example Confirm Message boxes
+
+```html
+<template>
+  <div>
+    <div class="mb-2">
+     <b-button @click="showMsgBoxOne">Simple msgBoxConfirm</b-button>
+     Return value: {{ String(boxOne) }}
+    </div>
+    <div class="mb-1">
+     <b-button @click="showMsgBoxTwo">msgBoxConfirm with options</b-button>
+     Return value: {{ String(boxTwo) }}
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    data() {
+      return {
+        boxOne: '',
+        boxTwo: ''
+      }
+    },
+    methods: {
+      showMsgBoxOne() {
+        this.boxOne = ''
+        this.$bvModal.msgBoxConfirm('Are you sure?')
+          .then(value => {
+            this.boxOne = value
+          })
+          .catch(err => {
+            // an error occured
+          })
+      },
+      showMsgBoxTwo() {
+        this.boxTwo = ''
+        this.$bvModal.msgBoxConfirm('Please confirm that you want to delete everything.', {
+          title: 'Please Confirm',
+          size: 'sm',
+          buttonSize: 'sm',
+          okVariant: 'danger',
+          okTitle: 'YES',
+          cancelTitle: 'NO',
+          footerClass: 'p-2',
+          hideHeaderClose: false,
+          centered: true
+        })
+          .then(value => {
+            this.boxTwo = value
+          })
+          .catch(err => {
+            // an error occured
+          })
+      }
+    }
+  }
+</script>
+
+<!-- msg-box-confirm.vue -->
+```
+
+### Message Box Notes
+
+- Message Boxes require `Promise` support in the browser. If targetting your app for older
+  browsers, such as IE 11, please include a polyfill that provides `Promise` support. If
+  `Promise` support is not detected, then the message box methods will immediately return
+  `undefined`.
+- Message Boxes are an extension of the `<b-modal>` component, and hence support the majority
+  of `<b-modal>` props (using <samp>camelCase</samp> format), with the exception of the
+  following props: `lazy`, `busy`, `visible`, `noStacking`, `okOnly`, `okDisabled`, and
+  `cancelDisabled`.
+- When a `title` (or `titleHtml`) _is not_ provided in the options, the header will not be shown.
+- When a `title` (or `titleHtml`) _is_ provided in the options, the header close button is not
+  shown by default. You can enable the header close button by setting `noHeaderClose: true` in
```

Comment:
@tmorehouse, two things here. 1) I think the option is `hideHeaderClose`, rather than `noHeaderClose` 2) Shouldn't it be setting it to `false` if you want it to show? That's what the example above does.

Reply:
Good catch